build/common/installer/scripts/tomlparser.rb

```diff
@@ -148,6 +149,16 @@ def populateSettingValuesFromConfigMap(parsedConfig)
       ConfigParseErrorLogger.logError("Exception while reading config map settings for kube event collection - #{errorStr}, using defaults, please check config map for errors")
     end
 
+    #Get PV kube-system enrichment setting
+    begin
+      if !parsedConfig[:log_collection_settings][:collect_kube_system_pv_metrics].nil? && !parsedConfig[:log_collection_settings][:collect_kube_system_pv_metrics][:enabled].nil?
+        @collectPVKubeSystemMetrics = parsedConfig[:log_collection_settings][:collect_kube_system_pv_metrics][:enabled]
```

Comment:
Please add telemetry to determine if kube-system pv metric collection is enabled for any given cluster. So that way, if we see more people enable it, we can turn it ON by default in the future.